app/components/primer/alpha/action_list_item.html.erb

```diff
@@ -0,0 +1,59 @@
+<%= render(Primer::BaseComponent.new(tag: :li, **@system_arguments)) do %>
+  <%= render(Primer::BaseComponent.new(tag: :span, **@content_arguments)) do %>
```

Comment:
For this tag, we want people to use `span` with an `onClick` for button type interactions, and `a href` for links. How can we pass those options down?

Reply:
Oh good call. Dotcom's `NavigationList` checks for the presence of an `href` attribute and uses an `a` tag if one is given. I can copy that logic over here. Sounds like we also need to expose an `on_click` attribute.